### PR TITLE
Tests electronics/electric_conductivity.py  #9943

### PR DESCRIPTION
**Pull request by juliaaragao**
### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Review by imSanko — CHANGES_REQUESTED** [review]
Nice changes

File: conversions/rec_to_pol.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import doctest
+import math
+
+
+def rec_to_pol(real: float, img: float) -> tuple:
```
Comment (cclauss) [inline conversions/rec_to_pol.py:5]:
```suggestion
def rectangular_to_polar(real: float, img: float) -> tuple[float, float]:
```
or `cartesian_to_polar`

File: conversions/rec_to_pol.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import doctest
```
Comment (cclauss) [inline conversions/rec_to_pol.py:1]:
```suggestion
```

File: conversions/rec_to_pol.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    """
+
+    mod = round(math.sqrt((real**2) + (img**2)), 2)
+
+    ang = round(math.degrees(math.atan2(img, real)), 2)
+    return (mod, ang)
+
+
+if __name__ == "__main__":
```
Comment (cclauss) [inline conversions/rec_to_pol.py:32]:
```suggestion
if __name__ == "__main__":
    import doctest

```

File: DIRECTORY.md [hunk 4]
```diff
@@ -169,6 +169,7 @@
   * [Prefix Conversions](conversions/prefix_conversions.py)
   * [Prefix Conversions String](conversions/prefix_conversions_string.py)
   * [Pressure Conversions](conversions/pressure_conversions.py)
+  * [Rec To Pol](conversions/rec_to_pol.py)
```
Comment (cclauss) [inline DIRECTORY.md:172]:
```suggestion
```
This is done automatically as discussed in CONTRIBUTING.md

File: conversions/rectangular_to_polar.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    >>> rec_to_pol(-1e-10,1e-10)
+    (0.0, 135.0)
+    >>> rec_to_pol(9.75,5.93)
+    (11.41, 31.31)
+    >>> rec_to_pol(10000,99999)
+    (100497.76, 84.29)
+    """
+
+    mod = round(math.sqrt((real**2) + (img**2)), 2)
+
```
Comment (cclauss) [inline conversions/rectangular_to_polar.py:25]:
```suggestion
```

File: conversions/rectangular_to_polar.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    (1.41, 135.0)
+    >>> rec_to_pol(-1,-1)
+    (1.41, -135.0)
+    >>> rec_to_pol(1e-10,1e-10)
+    (0.0, 45.0)
+    >>> rec_to_pol(-1e-10,1e-10)
+    (0.0, 135.0)
+    >>> rec_to_pol(9.75,5.93)
+    (11.41, 31.31)
+    >>> rec_to_pol(10000,99999)
```
Comment (cclauss) [inline conversions/rectangular_to_polar.py:20]:
```suggestion
    >>> rectangular_to_polar(5,-5)
    (7.07, -45.0)
    >>> rectangular_to_polar(-1,1)
    (1.41, 135.0)
    >>> rectangular_to_polar(-1,-1)
    (1.41, -135.0)
    >>> rectangular_to_polar(1e-10,1e-10)
    (0.0, 45.0)
    >>> rectangular_to_polar(-1e-10,1e-10)
    (0.0, 135.0)
    >>> rectangular_to_polar(9.75,5.93)
    (11.41, 31.31)
    >>> rectangular_to_polar(10000,99999)
```

**Review by cclauss — APPROVED** [review]
Nice!  A few changes for readability.

**Pull request merged**